### DISCUSSION TITLE
Drawing fabricjs rectangle coordinates (aCoords.br aCoords.tr, ...) while moving

### DISCUSSION THREAD

**Discussion by hathemi** [fabricjs/fabric.js#8862] [Q&A]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have searched and referenced existing issues, feature requests and discussions
- [X] I am filing a **FEATURE** request.

### Description

I'm trying to draw the x and y coordinates of the object rectangle while its moving. (x1, x2, y1, y2) as showing below
![Sans titre](https://user-images.githubusercontent.com/23275040/234127565-ce367d99-b50b-4cad-90c2-4d37e548659c.png)

**Any suggestion would be appreciated.**

### Current State

Only the the bounding box is drawing.

**Comment by melchiar** [comment]
This should help get you started. It's a script that overlays alignment guidelines on the canvas while moving an object.

https://github.com/fabricjs/fabric.js/blob/5.x/lib/centering_guidelines.js

**Reply by ShaMan123** [reply]
I don't really understand the use case but custom controls might be helpful as well
